settings/nitrofiles/languages/english.ini

```diff
@@ -102,7 +102,7 @@ DS_MAIN_MEMORY_I_CACHE=DS main memory i-cache
 WRAM_I_CACHE=WRAM i-cache
 BIOS_INTRO=BIOS intro
 
-DESCRIPTION_GBAR2_DLDIACCESS=Select which CPU to access the flashcard's DLDI driver. Select ARM7, if ARM9 causes a white screen crash.
+DESCRIPTION_GBAR2_DLDIACCESS=Select which CPU to access the flashcard's DLDI driver. Select ARM7 if ARM9 causes a white screen crash.
```

Comment:
to access -> accesses.

Reply:
Hmm, what do you think about `should access`? accesses is better than to access but I think that might be even better...
> Select which CPU should access the flashcard's DLDI driver.

Reply:
Yeah, much better.